src/components/Dashboard/Tiles/MaskChart/MaskPercTile.tsx

```diff
@@ -33,9 +33,7 @@ const mapStateToProps = (state: RootState) => {
   };
 };
 
-const MaskPercTile: React.FC<MaskPercTileProps> = (
-  props: MaskPercTileProps,
-): ReactElement => {
+const MaskPercTile = (props: MaskPercTileProps) => {
```

Comment:
nit: why was this removed?

Reply:
ditto for other files with similar edits

Reply:
Is it really necessary? I removed it cuz I felt it was redundant and it looks cleaner that without it.